packages/core/theme/src/components/select.ts

```diff
@@ -147,6 +167,12 @@ const select = tv({
         base: "w-full",
       },
     },
+    isClearable: {
+      true: {
+        input: "peer pr-6 rtl:pr-0 rtl:pl-6",
```

Comment:
don't use `rtl:*`. there use `start` and `end` for RTL. 

see https://tailwindcss.com/blog/tailwindcss-v3-3#simplified-rtl-support-with-logical-properties

Reply:
okay I will look into that.